src/constants/routes.ts

```diff
@@ -67,6 +68,13 @@ export const getBuildStatusIconLink = (patchId: string, buildVariant: string) =>
 export const getUserPatchesRoute = (userId: string): string =>
   `${paths.user}/${userId}/${PageNames.Patches}`;
 
+export const getLobsterTaskLink = (
+  logType: LogTypes,
+  taskId: string,
+  execution: number
+) =>
+  `${getLobsterURL()}/lobster/evergreen/task/${taskId}/${execution}/${logType}`;
```

Comment:
Logs aren't guaranteed to exist at this URL. I had thought that this was limited to just those tasks that are using buildlogger as their logger, but spot checking a few I think it might be more. For example neither https://logkeeper.mongodb.org/lobster/evergreen/task/evergreen_ubuntu1604_dist_01f29a346d16d5045f07a8494906088a049b481a_21_02_16_21_34_49/0/task#l=1 nor https://logkeeper.mongodb.org/lobster/evergreen/task/dot_net_driver_secure_tests__version~2.6_os~windows_64_topology~replicaset_auth~auth_ssl~ssl_test_net452_69618dd0a044a198f266f08c39a0b975a5ea6f27_21_02_12_18_48_23/0/task#l=1 work. I think in order for this to work we need to either fix the log fetching in lobster, or have a way for it to accept a url from which to download a log

Reply:
these logs seem to render when I log into evergreen with my credentials beforehand